src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import file.reader.FileReader;
+import file.reader.FileReaderImpl;
+import file.writer.FileWriter;
+import file.writer.FileWriterImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.converter.DataConverter;
+import service.converter.DataConverterImpl;
+import service.operation.BalanceOperation;
+import service.operation.OperationHandler;
+import service.operation.OperationStrategy;
+import service.operation.OperationStrategyImpl;
+import service.operation.PurchaseOperation;
+import service.operation.ReturnOperation;
+import service.operation.SupplyOperation;
+import service.report.ReportGenerator;
+import service.report.ReportGeneratorImpl;
+import service.shop.ShopService;
+import service.shop.ShopServiceImpl;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
According to the checklist, all services should be independent and called in the 'main()' method. The FileReader, DataConverter, OperationStrategy, ShopService, and ReportGenerator are correctly initialized and used here.